utils/functions

```diff
@@ -33,6 +33,13 @@
                  34sns-repair-1n-1f
                  35m0singlenode
                  36spare-reservation
+# 37protocol ST checks checksum of the BE structures during
+# developmental phase they are bound to fail so code
+# changes are required but ST has to be in disabled
+# state for now when the release is in GA phase we will
+# enable the ST again to ensure there will be no changes
+# in BE structures.
+                 37protocol
```

Comment:
Do not add this in the skip list.
Let it be enabled and instead please add an echo in the ST script file to say that this ST may fail if there are some local changes done to the BE structures and such failures can be safely ignored.

Reply:
Ok.

Reply:
Done